pkg/image/image.go

```diff
@@ -46,15 +53,17 @@ type UDF struct {
 }
 
 // Create makes the required call to UDF to
-func (u *UDF) Create(release, channel, arch string) (path string, err error) {
-	tmpFileName, _ := u.cli.ExecCommand("mktemp")
+func (u *UDF) Create(release, channel, arch string, ver int) (path string, err error) {
+	tmpDirName, _ := u.cli.ExecCommand("mktemp -d")
```

Comment:
I tried to use the dependency aliasing trick as least as possible, here the cli executor is injected, using ioutil.TempDir would require nasty mocks. I'll merge it as is, we can discuse further in another branch.